src/azure-cli/requirements.py3.windows.txt

```diff
@@ -120,7 +120,7 @@ pycomposefile==0.0.30
 pycparser==2.19
 PyGithub==1.55
 PyJWT==2.4.0
-pymsalruntime==0.14.2a1
+pymsalruntime==0.16.2a1
```

Comment:
MSAL 1.28.0 doesn't allow `pymsalruntime==0.16.2a1`:

```
> pip install msal[broker]==1.28.0 pymsalruntime==0.16.2a1
...
ERROR: Cannot install msal[broker]==1.28.0 and pymsalruntime==0.16.2a1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested pymsalruntime==0.16.2a1
    msal[broker] 1.28.0 depends on pymsalruntime<0.15 and >=0.13.2; (python_version >= "3.6" and platform_system == "Windows") and extra == "broker"
```

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.28.0/setup.cfg#L65

```py
    pymsalruntime>=0.13.2,<0.15; python_version>='3.6' and platform_system=='Windows'
```

We also need to bump MSAL when a new version is released.

Reply:
MSAL Python 1.28.1 was released yesterday.

Reply:
1.28.1 picked.